src/main/resources/payers/insurance_companies.csv

```diff
@@ -0,0 +1,10 @@
+﻿,id,name,address,city,state_headquartered,zip,phone,states_covered,services_covered,deductible,default_coinsurance,default_copay,monthly_premium,ownership

```

Comment:
Note: we need to add `TRICARE` insurance to this list for veterans only. Leave this as a TODO for some another pull request.